render/src/main/java/org/codice/imaging/nitf/render/NitfRenderer.java

```diff
@@ -218,6 +221,44 @@ private void renderJPEG2k(final ImageSegment imageSegment, final Graphics2D targ
             }
         }
 
+        /* If R, G, B are not present, the default displayable band
+        is the LU band
+         */
+        if (imageBands.isEmpty()) {
```

Comment:
This makes a lot more sense to me. Let me try this out and write a unit test to go along with it